app/views/errors/internal_server_error.html.haml

```diff
@@ -0,0 +1,6 @@
+%div#errors_page
```

Comment:
No need to declare `div` here. It's a `div` by default. But it's fine. Just for future ref :P ;)